### DISCUSSION TITLE
Finding a good layout - Pharmacy Room

### DISCUSSION THREAD

**Discussion by ageekhere** [CorsixTH/CorsixTH#2430] [General Diagnostics]
I am exploring good design layouts for each room, in this post I am focusing on the Pharmacy Room.

**The Goal**
Keeping the room as small as possible while keeping the doctor as happy as possible(avoid being unhappy and wanting a pay rise) while optimizing turnaround time for handyman plant watering and patient walk times.

Version 1.00
![Screenshot 2023-10-01 105057](https://github.com/CorsixTH/CorsixTH/assets/13173133/9eccab9f-aeef-4d0d-8602-a7f80d7a376c)

**Items**
1x Pharmacy Cabinet
1x Plant
1x Bin
1x Fire extinguisher
1x Radiator

**Summary**
The Pharmacy has a simple layout as it does not need many items, the main aspect is to get the Cabinet as close to the door as possible. In version 1.00 the patient opens the door and then moves onto the tile where the nurse can give the medication. Plant is next to door and the bin is in the walk path of the patient. The Fire extinguisher should cover the whole room with it being in the center. 1 Radiator should be able to heat the room.

Lets start the decision, can this design be improved and if so how?

**Comment by PaveMentman** [comment]
---
Pharmacy indeed is simple to optimize even when limited to north-side entrances.

Considering it is one of the most earliest treatment-rooms required for emergencies, having two Pharmacies near the helicopter-landing-pad is a must.
However, due to the benches needed near these rooms, this might prevent both of the Pharmacies to be as optimized especially in narrower corridors.

Here is a setup of mine:
Note!
Until more concrete or otherwise the rules have been more defined and documented,
the radiator and fire-extinguisher placement in my rooms are going to end being more or less "random" and instead primarily focused of patient-pathfinding.

![2023-10-01-16-47-24-459-CorsixTH-CorsixTH](https://github.com/CorsixTH/CorsixTH/assets/6709463/056916b5-eeee-4ddb-9ee3-961f175e2e68)
![2023-10-01-16-47-27-279-CorsixTH-CorsixTH](https://github.com/CorsixTH/CorsixTH/assets/6709463/bdf1f6e5-2447-448d-8265-371c094d4101)
---

**Reply by ageekhere** [reply]
If you look here https://github.com/CorsixTH/CorsixTH/discussions/2405 
The fire extinguisher has an effect within a radius of two tile for staff happiness. I do agree that this info needs to be more clear and added to the wiki.

Over all your designs should have the same patient turn around times are version 1.00.
Plant has easy access for handy man (hmm i now wonder what the plant radius is for staff happiness)
Bin is in the patient path.
